cristin-intermediate-storage/src/main/java/no/unit/nva/cristin/intermediate/storage/UpdateUnitsHandler.java

```diff
@@ -0,0 +1,61 @@
+package no.unit.nva.cristin.intermediate.storage;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestStreamHandler;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URI;
+import java.net.http.HttpClient;
+import nva.commons.core.Environment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.core.sync.RequestBody;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.PutObjectRequest;
+
+public class UpdateUnitsHandler implements RequestStreamHandler {
+
+    private static final Logger logger = LoggerFactory.getLogger(UpdateUnitsHandler.class);
+    private static final String BUCKET_NAME_ENV = "INTERMEDIATE_STORAGE_BUCKET_NAME";
+    private static final String BUCKET_KEY_FILE_NAME = "units-norway.json";
+    private static final String CRISTIN_API_URL_ENV = "CRISTIN_API_URL";
+    public static final String CRISTIN_BOT_FILTER_BYPASS_HEADER_NAME_ENV = "CRISTIN_BOT_FILTER_BYPASS_HEADER_NAME";
+    public static final String CRISTIN_BOT_FILTER_BYPASS_HEADER_VALUE_ENV = "CRISTIN_BOT_FILTER_BYPASS_HEADER_VALUE";
+    private final Environment environment;
+    private final S3Client s3Client;
+    private final HttpClient httpClient;
+
+    public UpdateUnitsHandler() {
```

Comment:
So we have to manually change the unit data inside this repository and then run the lambda in each repository by clicking on test run?

I have this strange feeling that it's simpler to just manually upload the files to the s3 bucket.